src/pages/index.tsx

```diff
@@ -1,25 +1,85 @@
-import type { NextPage } from "next";
-import { useRouter } from "next/router";
-import { useEffect } from "react";
+import { CommonButton, CommonWrapper } from '@components/common'
+import type { NextPage } from 'next'
+import { useRouter } from 'next/router'
+import { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import Splash from './splash'
 
 const Home: NextPage = () => {
-  const router = useRouter();
+  const router = useRouter()
+  const [isLoading, setIsLoading] = useState(false)
   useEffect(() => {
-    if (typeof window === "undefined") return;
-    window.localStorage.removeItem("GeoLocation");
-  }, []);
+    setIsLoading(true)
+    setTimeout(() => {
+      setIsLoading(false)
+    }, 1500)
```

Comment:
```suggestion
    setTimeout(() => {
      setIsLoading(false)
    }, 1500)
```
첫 렌더시에 isLoading이 true여야 하면 초기값을 true로 주고 useEffect에서 setIsLoading(true)는 없어도 되지 않나요?

Reply:
오 그게 더 간단하고 좋네요!!